README.md

```diff
@@ -1,2 +1,112 @@
-# aue
-Calculations for best- and worst-case scenario adult-use establishment distributions
+# Adult Use Establishments Proximity Analysis
+
+This repo contains code for identifying the best- and worst-case scenarios for the spatial distribution of Adult Use Establishments in NYC, given the restriction that no establishment be within 500ft of another establishment.
+
+## Background
+
+## Usage
+
+Run `./aue.sh` from the home directory. Results are found in the `output` directory. Each row contains the BBLs for a single solution. The number of columns is the total number of possible establishments. These counts are also output to the terminal.
+
+## Overview of Results
+
+Using the methodology outlined below, we have identified a best-case scenario of **208** lots that could feasibly have Adult Use Establishments, given the distance restrictions. The worst-case scenario, meaning the case where establishments are arranged in a way that limits the addition of new establishments, is **203** lots. Note that these numbers are the maximum *possible* establishments. In reality, not all lots are currently vaccant.
+
+These numbers are dependent on the input data, however, and there is ongoing work vetting the input lots as realistic locations for an establishment (i.e. there is sufficient street-frontage and access). In the future, this workflow can be re-run with refined input data to improve the estimate of best- and worst-case scenarios, simply by modifying the path for the input buffered lots.
+
+# Methodology
+
+We approached this problem by creating a matrix of possible co-existing establishments. Simplifying the distance calculations to a binary flag makes our calculations much easier. 
+
+### Data loading and preparation
+
+**Input data**:
+
+The input data for this analysis is a shapefile of lots, buffered to 500ft. See the image below for an example of what this kind of data looks like.
+
+![input-data](https://github.com/NYCPlanning/aue/blob/master/input-data.png "Buffered Lots")
+
+
+**Creating a "possibility matrix"**: 
+
+Using the buffers in the input data, we can create a square matrix that represents whether an establishment on one lot can coexist with an establishment on another lot. If two establishments can exist together, the value is 1. If they cannot, the value is 0. The diagonals are null.
+
+Note that this matrix is always square, and is always symetrical. To create this matrix, we test whether or not two buffers intersect using postgres,
+then pivot the pair-wise intersection data using python. Below is a simple example of a lot arrangement and how the resulting possibility matrix would look.
```

Comment:
I think "using postgres" and "using python" are unnecessary details, but if you decide to keep them change "postgres" to "postgis"